geomstats/geometry/fiber_bundle.py

```diff
@@ -33,8 +39,10 @@ class FiberBundle(Manifold):
         group. Either dim, base or group must be given as input.
     """
 
-    def __init__(self, total_space, base=None, group=None, group_action=None,
-                 dim=None, **kwargs):
+    def __init__(
```

Comment:
Yes, we need to decide if we use typing everywhere! @MariusGuerard was also thinking about this.